v2/cli/init_command.go

```diff
@@ -118,17 +125,23 @@ func initialize(c *cli.Context) error {
 		}
 
 		packagePath := path.Join(".brigade", "package.json")
-		if overwrite, err = checkFileOverwrite(packagePath); err != nil {
+		if fileWrite, overwrite, err =
+			checkFileOverwrite(packagePath); err != nil {
 			return err
 		}
-		if overwrite {
+		if fileWrite {
 			if err = ioutil.WriteFile(
 				packagePath,
 				packageBytes,
 				0644,
 			); err != nil {
 				return err
 			}
+			if overwrite {
+				fmt.Println("Overwriting .brigade/package.json...")
```

Comment:
Can we use `packagePath` instead of writing out the path again here? (Especially since forward slash may be inaccurate on a Windows system.) Same comment applies numerous other places.